src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -722,6 +722,37 @@ void setTL0Idx(int newTL0Idx)
         }
 
         /**
+         * Given a list of available quality indices and a target index, find
+         * the highest available quality index <= targetIdx
+         * @param availableQualityIndices the available quality indices
+         * @param targetIdx the target index
+         * @return the highest quality index <= targetIdx
+         * //FIXME(brian): this could be static, but can't add a static method
+         * to the inner class.  should it be put elsewhere and made a static
+         * helper method? (the outer class?)
+         */
+        private int findMaximumQualityIndex(int[] availableQualityIndices, int targetIdx)
+        {
+            int maxIndex = -1;
+            if (availableQualityIndices != null)
+            {
+                for (int i = 0; i < availableQualityIndices.length; ++i)
```

Comment:
talked with george and clarified a few things...we do want it with the old behavior.  couple other tweaks i want to make to this as well so will work on those.